nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -103,12 +104,20 @@ class ICA_AROMA(CommandLine):
 
     def _list_outputs(self):
         outputs = self.output_spec().get()
-        out_dir = os.path.abspath(self.inputs.out_dir)
-        outputs['out_dir'] = out_dir
-
+        if isdefined(self.inputs.out_dir):
```

Comment:
I think I need the isdefined module for this line to run, please let me know if I'm mistaken.